hw_32_form/js/validator.js

```diff
@@ -0,0 +1,90 @@
+import validationError from "./validation-error.js";
+
+export const Validator = {
+    errors: {},
+    emailFormat: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+    valuesToCompare: {},
+
+    validators: {
+        isNotEmpty: {
+            validate: (value) => value !== '',
+            message: "The field can't be blank",
+            errorType: 'required'
+        },
+        isNumber: {
+            validate: (value) => !isNaN(value) && Number.isInteger(+value),
+            message: "The field must be an integer",
+            errorType: 'number'
+        },
+        length({ minLength = 1, maxLength = 50 }) {
```

Comment:
length is a reserved word for js, so you have to think of something else here. (i.e. `isBetween`)